### ISSUE TITLE
audit-avt1-guidelines-motion-choreography: inline frame title attr.

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon-website#1505]
## Environment:
> Operating system: Mac
> Browser: Chrome
> AVT :one:
> Page: [Guidelines - Motion: Choreography](https://www.carbondesignsystem.com/guidelines/pictograms/usage)
> This bug affects 15 total issues

## Detailed Description:
Inline frame does not have a 'title' attribute

<img width="1680" alt="Screen Shot 2020-08-04 at 12 56 54 PM" src="https://user-images.githubusercontent.com/32720851/89324654-a068ac00-d655-11ea-915e-2193cdbeaa47.png">

## What to do:
Use the "title" attribute of the <iframe> element to provide a unique, non-empty title that describes its contents.

## Why is this important?
An inline frame’s title provides a label for the frame that describes its content to the user. With this information a user can determine which frame to enter and explore in detail.
